source/features/new-or-deleted-file.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '../feature-manager.js';
+import observe from '../helpers/selector-observer.js';
+
+function add(filename: HTMLAnchorElement): void {
+	const list = select('ul[aria-label="File Tree"]');
```

Comment:
This feature used to use this dropdown:

<img width="505" alt="Screenshot 4" src="https://github.com/refined-github/refined-github/assets/1402241/37955dfa-3163-4f3d-8eb0-5687e748b72f">

but it only appears on PRs _and_ it's actually been relegated to small screens, because the whole dropdown is hidden/duplicate of the sidebar on large screens.

It seems that the sidebar is actually part of the original HTML, so we don't even need to wait for it to load (at least some times)

Reply:
I ended up _not_ using the API because I figured we'd still have trouble figuring out the exact changes/api to consider in case of single commits and multi-PR-commits. The sidebar is recent enough, hopefully it will last a while.